### PR TITLE
Fix: typos

### PR DESCRIPTION
**Pull request by omahs**
Fix: typos

### CODE REVIEW

**Comment by The-3D** [comment]
can we point this to the 3.0.1 branch?

**Comment by omahs** [comment]
@The-3D Is it correct branch now? aave:feat/3.0.1

File: contracts/protocol/pool/DefaultReserveInterestRateStrategy.sol [hunk 1]
```diff
@@ -191,8 +191,8 @@ contract DefaultReserveInterestRateStrategy is IReserveInterestRateStrategy {
   }
 
   /// @inheritdoc IReserveInterestRateStrategy
-  function calculateInterestRates(DataTypes.CalculateInterestRatesParams calldata params)
-    external
+  function calculateInterestRates(DataTypes.CalculateInterestRatesParams memory params)
+    public
```
Comment (kyzia551) [inline contracts/protocol/pool/DefaultReserveInterestRateStrategy.sol:195]: @miguelmtzinf is it needed?

Comment (miguelmtzinf) [reply contracts/protocol/pool/DefaultReserveInterestRateStrategy.sol:195]: Github's diff is broken. These changes are already part of the codebase

**Review by miguelmtzinf — COMMENTED** [review]
Thanks @omahs for your work!

**Pull request merged**